SourceX/dx.cpp

```diff
@@ -205,6 +205,21 @@ void BltFast(DWORD dwX, DWORD dwY, LPRECT lpSrcRect)
 	bufferUpdated = true;
 }
 
+/**
+ * @brief Limit FPS to lower CPU when not waiting for V-Blank
+ */
+void LimitFrameRate()
+{
+	static uint32_t frameDeadline;

```

Comment:
nit: These should be `std::uint32_t` and `std::fmod` in C++

Reply:
This gives me a build error saying uint32_t isn't part of std

Reply:
`#include <cstdint>`